models/reviews-model.js

```diff
@@ -28,13 +28,54 @@ exports.updateVotesById = (reviewId, incVotes) => {
     });
 };
 
-// TASK 8
-exports.fetchAllReviews = () => {
-  return db
-    .query(
-      `SELECT owner, title,  reviews.review_id, category,  review_img_url, reviews.created_at, reviews.votes, COUNT(comments.review_id) AS comment_count FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;`
-    )
-    .then((data) => {
-      return data.rows;
-    });
+// TASK 8 and 11
+exports.fetchAllReviews = (
+  sort_by = "created_at",
+  order = "DESC",
+  category
+) => {
+  const validSortBy = [
+    "category",
+    "title",
+    "created_at",
+    "votes",
+    "owner",
+    "review_id",
+    "comment_count",
+  ];
+
+  const validCategories = [
```

Comment:
It's perfectly valid to check the sort by and the order by. 

However checking the categories like this is very brittle!
What if more topics are added in the future? Also what if there are difference between the test and dev data?

Do we have to check before running the query? What happens if we don't?